src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.macios.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System.Collections.ObjectModel;
+using System.Runtime.InteropServices;
+using CommunityToolkit.Maui.Core.Extensions;
+using CoreGraphics;
+using Foundation;
+using Microsoft.Maui.Platform;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+public partial class MauiDrawingView : UIView
+{
+	readonly UIBezierPath currentPath = new();
+	readonly List<UIScrollView> scrollViewParents = new();
+
+	CGPoint previousPoint;
+	MauiDrawingLine? currentLine;
+
+	/// <summary>
+	/// Line color
+	/// </summary>
+	public UIColor LineColor { get; set; } = Colors.Black.ToPlatform();
+
+	/// <summary>
+	/// Line width
+	/// </summary>
+	public NFloat LineWidth { get; set; } = new NFloat(5);
```

Comment:
Not something that will change the approval of this PR, but do we have to do this, `new NFloat` is very ugly >.<''

Reply:
as for me it looks cleaner then `(NFloat)5`

Reply:
Yeah, like it's not our fault, but I would like to have an implicit conversion, then we could just do `public NFloat LineWidth { get; set; } =  5;`

Reply:
It's just me thinking aloud

Reply:
I was almost sure it doesn't work. I got an error like this:
![image](https://user-images.githubusercontent.com/33021114/159141843-c4c98617-fba0-4b5b-a33e-87c7d1940c46.png)
fixed!